src/api/types/index.tsx

```diff
@@ -0,0 +1,93 @@
+import DefaultFieldKey from "../../constants/DefaultFieldKey";
+import EnrolmentStatus from "../../constants/EnrolmentStatus";
+import { Class, Session, Family, Student, DynamicField } from "../../types";
+
+export type ClassListResponse = Pick<Class, "id" | "name">;
+
+export type SessionListResponse = Pick<Session, "id" | "season" | "year">;
+
+export type SessionDetailResponse = Session & {
+  classes: ClassListResponse[];
+  families: FamilyListResponse[];
+};
+
+export type ClassDetailResponse = Class & {
+  families: FamilyListResponse[];
+};
```

Comment:
nit: can we move this above SessionListResponse to keep the class types together?